packages/jumpstarter-driver-flashers/jumpstarter_driver_flashers/driver.py

```
@@ -177,7 +182,11 @@ async def get_initram_filename(self) -> str | None:
     async def get_dtb_filename(self) -> str:
         """Return the dtb filename"""
         manifest = await self.get_flasher_manifest()
-        return Path(manifest.get_dtb_file(self._use_dtb)).name
+        dtb_file = manifest.get_dtb_file(self.variant)
+        if dtb_file:
+            return Path(manifest.get_dtb_file(self.variant)).name
+        else:
+            return ""
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)